lib/factor/factorQuadratic.js

```diff
@@ -192,22 +197,62 @@ function factorPerfectSquare(node, symbol, aValue, bValue, cValue, negate) {
   return Node.Status.noChange(node);
 }
 
-// Will factor the node if it's in the form of x^2 + bx + c (i.e. a is 1), by
-// applying the sum product rule: finding factors of c that add up to b.
-// e.g. x^2 + 3x + 2 -> (x + 1)(x + 2)
+// Will factor the node if it's in the form of x^2 + bx + c, by
+// applying the sum product rule: finding factors of a*c that add up to b.
+// e.g. x^2 + 3x + 2 -> (x + 1)(x + 2) or
+// or   2x^2 + 5x + 3 -> (2x - 1)(x + 3)
 function factorSumProductRule(node, symbol, aValue, bValue, cValue, negate) {
-  if (aValue === 1 && bValue && cValue) {
-    // try sum/product rule: find a factor pair of c that adds up to b
-    const factorPairs = ConstantFactors.getFactorPairs(cValue, true);
+  if (bValue && cValue) {
+    const gcd = math.gcd(aValue, bValue, cValue);
+    aValue = aValue/gcd;
+    bValue = bValue/gcd;
+    cValue = cValue/gcd;
+
+    const product = aValue * cValue;
+
+    // try sum/product rule: find a factor pair of a*c that adds up to b
+    const factorPairs = ConstantFactors.getFactorPairs(product, true);
     for (const pair of factorPairs) {
       if (pair[0] + pair[1] === bValue) {
+        // bValue is split into pair[0] + pair[1]
+        // group the terms
+        const firstGroup = [aValue, pair[0]];
+        const secondGroup = [cValue, pair[1]];
+        const firstGroupGcd = math.gcd(...firstGroup);
+        const secondGroupGcd = math.gcd(...secondGroup);
+
+        // factor the first group: e.g. 2x^2 + 4x -> 2x * (x + 1)
+        const firstCoeffValue = Node.Creator.constant(firstGroupGcd);
+        const firstCoeff = Node.Creator.polynomialTerm(
+          symbol, null, firstCoeffValue);
+
+        const firstTermInGroupCoeff = Node.Creator.constant(aValue/firstGroupGcd);
```

Comment:
Yea, I played around with something like that too, but I agree it feels just as confusing. A comment at the top that explains it all, could be the trick though. Let me try that.